src/test/java/com/crud/tasks/mapper/TrelloMapperTest.java

```diff
@@ -0,0 +1,155 @@
+package com.crud.tasks.mapper;
+
+import com.crud.tasks.domain.*;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.util.stream.Collectors.toList;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class TrelloMapperTest {
+
+    @Autowired
+    TrelloMapper trelloMapper;
+
+    private List<TrelloList> initTrelloList() {
+        List<TrelloList> trelloList = new ArrayList<>();
+        trelloList.add(new TrelloList("1", "test1", false));
+        trelloList.add(new TrelloList("2", "test2", false));
+        trelloList.add(new TrelloList("3", "test3", true));
+        return trelloList;
+    }
+
+    private List<TrelloListDto> initTrelloListDto() {
+        List<TrelloListDto> trelloListDto = new ArrayList<>();
+        trelloListDto.add(new TrelloListDto("1", "test1", false));
+        trelloListDto.add(new TrelloListDto("2", "test2", false));
+        trelloListDto.add(new TrelloListDto("3", "test3", true));
+        return trelloListDto;
+    }
+
+    private List<TrelloBoard> initTrelloBoards() {
+        List<TrelloBoard> trelloBoards = new ArrayList<>();
+        trelloBoards.add(new TrelloBoard("1", "Test1", initTrelloList()));
+        trelloBoards.add(new TrelloBoard("2", "Test2", initTrelloList()));
+        trelloBoards.add(new TrelloBoard("3", "Test3", initTrelloList()));
+        return trelloBoards;
+    }
+
+    private List<TrelloBoardDto> initTrelloBoardsDto() {
+            List<TrelloBoardDto> trelloBoardsDto = new ArrayList<>();
+            trelloBoardsDto.add(new TrelloBoardDto("1", "Test1", initTrelloListDto()));
+            trelloBoardsDto.add(new TrelloBoardDto("2", "Test2", initTrelloListDto()));
+            trelloBoardsDto.add(new TrelloBoardDto("3", "Test3", initTrelloListDto()));
+            return trelloBoardsDto;
+    }
+
+    @Test
+    public void mapToList() {
```

Comment:
co nam mowi taka nazwa metody?  kompletnie nic. proponuje ponadto zaczynac nazwy metod testowych od should lub test